src/vivarium_gates_nutrition_optimization/constants/data_values.py

```diff
@@ -20,7 +20,7 @@ class _Durations(NamedTuple):
 
 
 DURATIONS = _Durations()
-draw_count = vi_globals.NUM_DRAWS
```

Comment:
Delete the import if you're no longer using it. (in fact, shouldn't this still break?)